content/IA-update_connections/Overview.textile

```diff
@@ -0,0 +1,293 @@
+---
```

Comment:
We don't capitalize filenames generally. Also, according to Miro, this file should be `index.textile`, in the `connect` folder.

Reply:
Done